ui/src/App.tsx

```diff
@@ -25,124 +28,111 @@ type Props = {
 };
 
 const App: React.FC<Props> = () => {
-  const [echoMsg, setEchoMsg] = useState("this should travel to server and back to UI");
-  const [isCz, setIsCz] = useState(true);
 
-  const testEcho = async () => {
-
-    const getArticle = withFetch<TAction<{ echoMsg: string }, { echoMsg: string }>>({
-      route: "test/echo",
-      method: "POST",
-    });
-
-    try {
-      const test = await getArticle({ echoMsg });
-      alert(`response success ${JSON.stringify(test)}`);
-    } catch (e) {
-      alert(e);
-    }
-  };
-
-  const testError = async () => {
-
-    const getArticle = withFetch<TAction<{ echoMsg: string }, { echoMsg: string }>>({
-      route: "test/error",
-      method: "DELETE",
-    });
-
-    try {
-      const test = await getArticle({ echoMsg });
-      alert(`response success ${JSON.stringify(test)}`);
-    } catch (e) {
-      if (e instanceof ErrorCode)
-        alert(`coded error ${e.code}:\n ${e.message}`);
-      else
-        alert(`error without code:\n ${e.message}`);
-    }
-  };
+  const [isCz, setIsCz] = useState(true);
 
   return (<>
     <LanguageContext.Provider value={isCz ? ELanguages.cz : ELanguages.en}>
-      {/*Change language icons*/}
-      <div>
-        <button
-          onClick={() => setIsCz(!isCz)}
-          style={{ position: "fixed", fontSize: "3em", right: 0 }}
-        >
-          <LocText
-            cz="cz"
-            en="en"
-          />
-        </button>
-      </div>
-
-      <div>
-        <LocText
-          en="Send echo messagee"
-          cz="Poslat testovací zprávu"
-        />
-        <input value={echoMsg} onChange={e => setEchoMsg(e.target.value)}></input>
-        <button onClick={testEcho}>Send</button>
-      </div>
-      <div>Send error message
-          <button onClick={testError}>Send</button>
-      </div>
 
       <Router>
+
+        {/* CHANGE LANGUAGE ICONS */}
+        <ul className="language-menu">
+          <li className="nav-item">
+            <button onClick={() => setIsCz(!isCz)}>
```

Comment:
Clicking eng icon switches languages. We expect language to be eng after clicking eng icon.
Change `setIsCz(!isCz)` into `setIsCz(false)`